File: Examples/Features/Formatting/Fonts/FontSize.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace GrapeCity.Documents.Excel.Examples.Features.Formatting.Fonts
{
    public class FontSize : ExampleBase
    {
        public override void Execute(GrapeCity.Documents.Excel.Workbook workbook)
        {
            IWorksheet worksheet = workbook.Worksheets[0];

            worksheet.Range["A1"].Value = "Font size is 15";
            worksheet.Range["A1"].Font.Size = 15;
        }
    }
}

File: Examples.VB/Features/Formatting/Fonts/FontSize.vb
Namespace Features.Formatting.Fonts
    Public Class FontSize
        Inherits ExampleBase
        Public Overrides Sub Execute(workbook As Excel.Workbook)
            Dim worksheet As IWorksheet = workbook.Worksheets(0)
            worksheet.Range!A1.Value = "Font size is 15"
            worksheet.Range!A1.Font.Size = 15
        End Sub
    End Class
End Namespace
